config/src/config/netbench.rs

```diff
@@ -13,6 +13,7 @@ pub struct NetbenchConfig {
     pub enable_direct_send_testing: bool, // Whether or not to enable direct send test mode
     pub direct_send_data_size: usize,     // The amount of data to send in each request
     pub direct_send_per_second: u64,      // The interval (microseconds) between requests
+    pub direct_send_ramp_up_ms: Option<u64>, // The interval (milliseconds) for each ramp up phase
```

Comment:
I wish this was more explanitory, e.g.
"For three of this time period the send rate will be 1/3, 1/2, 2/3 of full rate, then full rate."